mk/spksrc.service.mk

```diff
@@ -222,41 +222,64 @@ ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
 $(DSM_CONF_DIR)/privilege:
 	$(create_target_dir)
 	@jq -n '."defaults" = {"run-as": "package"}' > $@
-else
-ifeq ($(strip $(SERVICE_EXE)),)
-$(DSM_CONF_DIR)/privilege: $(SPKSRC_MK)spksrc.service.privilege-installasroot
-	@$(dsm_script_copy)
-else
-$(DSM_CONF_DIR)/privilege: $(SPKSRC_MK)spksrc.service.privilege-startasroot
-	@$(dsm_script_copy)
-endif
-endif
+	@$(MSG) "Creating $@"
+	@$(MSG) '(privilege) run-as: package'
+	@$(MSG) "(privilege) DSM >= 7 $(DSM_CONF_DIR)/privilege"
 # Apply variables to privilege file
-ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
 ifneq ($(strip $(GROUP)),)
 # Creates group but is different from the groups the user can create, they are invisible in the UI an are only usefull to access another packages permissions (ffmpeg comes to mind)
 # For DSM7 I recommend setting permissions for individual packages (System Internal User)
 # or use the shared folder resource worker to add permissions, ask user from wizard see transmission package for an example
 	@jq --arg packagename $(GROUP) '."join-pkg-groupnames" += [{$$packagename}]' $@ 1<>$@
```

Comment:
No package is using it, yet maybe we can rename this?  (actually I don't know 100% what it is useful for) afaik the package can add another package here to allow it to access its files (I just don't remember which package needs to define it). DSM7 only.

Reply:
Actually I don't see it documented any more. Maybe it was only in the DSM7 preview?

Reply:
If I get this right, a good use-case that comes to mind would be for my `spksrc-kernel` toolkit where a `sc-kernel` group could be created and `synokernel-usbserial`, `synokernel-usbwifi`, `synokernel-linuxtv` could then make use of it?

If that is the case, I guess we should elaborate a little more and allow it on DSM6 somehow if possible to facilitate future migrations as I would use that right away.

Reply:
It is implemented by using package names. Not groups. I know I fell for that one too.